### PR TITLE
阶段 3 提案评审

### LINKED ISSUE #8616: 阶段 3 提案评审
- 原文链接：[Stage 3 Proposal Reviews](https://github.com/tc39/how-we-work/blob/master/stage-3-review.md)
- Markdown文件：[文件地址](https://github.com/tc39/how-we-work/blob/master/stage-3-review.md)
- PR 地址：
- 文章分类：**前端**
- **注意：文件位置在 `article/ECMA-TC39` 文件夹中，不是之前的 `article/2021` 了。**

---

- 翻译时间：5 天
- 校对时间：2 天
- 翻译积分：3 分
- 校对积分：1 分
- [积分有什么用](https://github.com/xitu/gold-miner/wiki)
- [ECMAScript翻译指南（2021.5）](https://github.com/xitu/gold-miner/wiki/ECMAScript%E7%BF%BB%E8%AF%91%E6%8C%87%E5%8D%97%EF%BC%882021.5%EF%BC%89)

### PR DESCRIPTION
**Pull request by Usualminds**
译文翻译完成，resolve #8616

### CODE REVIEW

**Comment by Usualminds** [comment]
@lsvih 翻译完成

**Comment by KimYangOfCat** [comment]
@lsvih 校对认领

File: article/ECMA-TC39/Stage-3-Proposal-Reviews.md [hunk 1]
```diff
@@ -2,36 +2,35 @@
...
+# 阶段 3 提案评审
 
-According to [the TC39 process document](https://tc39.es/process-document), to advance from Stage 2 to Stage 3, reviewers and the editor group must sign off on the spec text. This document describes the Stage 3 review process.
+根据 [TC39 流程文档](https://tc39.es/process-document)，提案要从阶段 2 过渡到阶段 3，必须要评审人员和编辑小组在规范文档上签字。本文主要介绍阶段 3 的审核过程。
 
-## Choosing reviewers
+## 选择评审人
 
-Reviewers for Stage 3 volunteer as such in TC39 meetings when discussing a proposal. If the proposal is ready, the champion can ask the committee for reviewers at the same meeting as it advances to Stage 2. Sizable proposals are encouraged to have multiple reviewers. Different reviewers can provide different perspectives.
+作为志愿者，阶段 3 的评审人员在 TC39 会议中讨论提案。如果提案准备就绪，提案的支持者可以在提案进入第二阶段的同一次会议上，要求委员会进行提案审查。规模较大的提案鼓励有多位评审人参与。因为不同的评审人员可以提供不同的视角。
```
Comment (KimYangOfCat) [inline article/ECMA-TC39/Stage-3-Proposal-Reviews.md:14]:
```suggestion
作为志愿者，阶段 3 的评审人员在 TC39 会议中讨论提案。如果提案准备就绪，提案发起人可以在提案进入第二阶段的同一次会议上，要求委员会进行提案审查。规模较大的提案鼓励有多位评审人参与。因为不同的评审人员可以提供不同的视角。
```

File: article/ECMA-TC39/Stage-3-Proposal-Reviews.md [hunk 2]
```diff
@@ -2,36 +2,35 @@
...
 
-Reviewer feedback is tracked in a GitHub issue in the proposal repository ([example](https://github.com/tc39/proposal-regexp-unicode-property-escapes/issues/3)). It may also be useful to send out a reminder email ahead of the meeting to the reviewers.
+评审人员的反馈将会在 GitHub 议题（issue）的提案库中被追踪（[示例](https://github.com/tc39/proposal-regexp-unicode-property-escapes/issues/3)）。在会议开始之前，给评审人员发一封邮件也是有帮助的。
```
Comment (KimYangOfCat) [inline article/ECMA-TC39/Stage-3-Proposal-Reviews.md:16]:
```suggestion
评审人员的反馈将会在提案库的 GitHub 议题（issue）中被追踪（[示例](https://github.com/tc39/proposal-regexp-unicode-property-escapes/issues/3)）。在会议开始之前，给评审人员发一封邮件也是有帮助的。 
```

File: article/ECMA-TC39/Stage-3-Proposal-Reviews.md [hunk 3]
```diff
@@ -2,36 +2,35 @@
...
 
-## Choosing reviewers
+## 选择评审人
 
-Reviewers for Stage 3 volunteer as such in TC39 meetings when discussing a proposal. If the proposal is ready, the champion can ask the committee for reviewers at the same meeting as it advances to Stage 2. Sizable proposals are encouraged to have multiple reviewers. Different reviewers can provide different perspectives.
+作为志愿者，阶段 3 的评审人员在 TC39 会议中讨论提案。如果提案准备就绪，提案的支持者可以在提案进入第二阶段的同一次会议上，要求委员会进行提案审查。规模较大的提案鼓励有多位评审人参与。因为不同的评审人员可以提供不同的视角。
 
-Reviewer feedback is tracked in a GitHub issue in the proposal repository ([example](https://github.com/tc39/proposal-regexp-unicode-property-escapes/issues/3)). It may also be useful to send out a reminder email ahead of the meeting to the reviewers.
+评审人员的反馈将会在 GitHub 议题（issue）的提案库中被追踪（[示例](https://github.com/tc39/proposal-regexp-unicode-property-escapes/issues/3)）。在会议开始之前，给评审人员发一封邮件也是有帮助的。 
+
```
Comment (KimYangOfCat) [inline article/ECMA-TC39/Stage-3-Proposal-Reviews.md:17]:
```suggestion
```
这里多加了一行导致后面顺序错乱

File: article/ECMA-TC39/Stage-3-Proposal-Reviews.md [hunk 4]
```diff
@@ -2,36 +2,35 @@
...
+在评审 TC39 提案时，请仔细阅读规范说明、README 和其他文档。在评审过程中，下面的几个问题点可能对你有帮助：
 
-When reviewing a TC39 proposal, read the specification text, README and other documentation carefully. Some questions which may be useful to consider in a review:
+- 该提案是否与其提案动机相关联？
+- 该提案的语言特性是否直观和易于学习？
+- 该提案和其他 JavaScript 语言特性（当前的特性和提案中的特性，即横切关注点）是否匹配？ 这两者之间的交互是否出人意料或者令人感觉陌生？
+- 该提案是否有意义，或者将它作为更高一级提案或低一级提案更具匹配一些？
+- 规范说明书是否完全覆盖了提案的各个方面，还是说存在一些未加以说明的地方？
+- 规范文本的逻辑性和一致性和其他 JavaScript 规范是否匹配？
+- 所有的提案细节和周边案例是不是都有合理的动机？
```
Comment (KimYangOfCat) [inline article/ECMA-TC39/Stage-3-Proposal-Reviews.md:28]:
```suggestion
- 所有的提案细节和边界案例是不是都有合理的动机？
```

**Review by KimYangOfCat — COMMENTED** [review]
@lsvih 校对完毕
翻译质量很好，赞

**Comment by lsvih** [comment]
@KimYangOfCat 感谢校对

File: article/ECMA-TC39/Stage-3-Proposal-Reviews.md [hunk 5]
```diff
@@ -2,36 +2,35 @@
...
 
-- Does the proposal address its stated motivation?
-- Is this language feature intuitive and learnable?
-- Does the proposal fit in well when combined with other JavaScript language features (both current and proposed, a.k.a. "cross-cutting concerns")? Will the interactions between features be surprising or strange?
-- Does the proposal scope make sense, or would this make more sense as a larger or smaller proposal?
-- Does the specification text completely cover every aspect of the proposal, or are some things unstated?
-- Is the specification text logical and consistent, matching the rest of the JavaScript specification?
-- Do all of the details and edge cases seem reasonably motivated?
+向提案支持小组提供你的反馈、GitHub 中的议题文件或者提出修改意见的合并请求。
```
Comment (PassionPenguin) [inline article/ECMA-TC39/Stage-3-Proposal-Reviews.md:30]:
```suggestion
向提案支持小组提供你的反馈，在 GitHub 中提出议题或者提出修改意见的合并请求。
```

嘛，另外两个都是动宾就这个有些突兀，显然就有些问题啦！

这里的 file 应该指的是**提出**：）

> • submit (a legal document, application, or charge) to be placed on record by the appropriate authority: criminal charges were filed against the firm | [no object] : the company had filed for bankruptcy.

File: article/ECMA-TC39/Stage-3-Proposal-Reviews.md [hunk 6]
```diff
@@ -2,36 +2,35 @@
...
+ 
+## 进行提案评审
 
-## Performing a review
+在评审 TC39 提案时，请仔细阅读规范说明、README 和其他文档。在评审过程中，下面的几个问题点可能对你有帮助：
 
-When reviewing a TC39 proposal, read the specification text, README and other documentation carefully. Some questions which may be useful to consider in a review:
+- 该提案是否与其提案动机相关联？
+- 该提案的语言特性是否直观和易于学习？
+- 该提案和其他 JavaScript 语言特性（当前的特性和提案中的特性，即横切关注点）是否匹配？ 这两者之间的交互是否出人意料或者令人感觉陌生？
```
Comment (PassionPenguin) [inline article/ECMA-TC39/Stage-3-Proposal-Reviews.md:24]:
```suggestion
- 该提案和其他 JavaScript 语言特性（当前的特性和提案中的特性，即横切关注点）是否匹配？这两者之间的交互是否出人意料或者令人感觉陌生？
```

File: article/ECMA-TC39/Stage-3-Proposal-Reviews.md (hunk 3 above)
Comment (PassionPenguin) [reply article/ECMA-TC39/Stage-3-Proposal-Reviews.md:17]: 其实没有……

是 GitHub 错位了罢了（

按照 Markdown Syntax，标题前后是需要空行的（

File: article/ECMA-TC39/Stage-3-Proposal-Reviews.md [hunk 7]
```diff
@@ -2,36 +2,35 @@
...
 > * 本文永久链接：[https://github.com/xitu/gold-miner/blob/master/article/ECMA-TC39/Stage-3-Proposal-Reviews.md](https://github.com/xitu/gold-miner/blob/master/article/ECMA-TC39/Stage-3-Proposal-Reviews.md)
-> * 译者：
+> * 译者：[Usualminds](https://github.com/Usualminds)
 > * 校对者：
 
-# Stage 3 Proposal Reviews
+# 阶段 3 提案评审
 
-According to [the TC39 process document](https://tc39.es/process-document), to advance from Stage 2 to Stage 3, reviewers and the editor group must sign off on the spec text. This document describes the Stage 3 review process.
+根据 [TC39 流程文档](https://tc39.es/process-document)，提案要从阶段 2 过渡到阶段 3，必须要评审人员和编辑小组在规范文档上签字。本文主要介绍阶段 3 的审核过程。
```
Comment (PassionPenguin) [inline article/ECMA-TC39/Stage-3-Proposal-Reviews.md:10]:
```suggestion
根据 [TC39 流程文档](https://tc39.es/process-document)，提案要从阶段 2 过渡到阶段 3，必须要评审人员和编辑小组同意这一份规范文档的提升。本文主要介绍阶段 3 的审核过程。
```

嗷：

> sign off on
> (sign off on something) US informal give one's approval to something: it was hard to get celebrities to sign off on those issues.

File: article/ECMA-TC39/Stage-3-Proposal-Reviews.md (hunk 1 above)
Comment (PassionPenguin) [reply article/ECMA-TC39/Stage-3-Proposal-Reviews.md:14]:
```suggestion
阶段 3 的评审人员需要自愿提供评审，即在 TC39 会议中讨论提案。如果提案准备就绪，提案的支持者可以在提案进入阶段 2 的同一次会议上，要求委员会进行提案审查。规模较大的提案鼓励多位评审人一起参与，因为不同的评审人员可以提供不同的视角下的评审。
```

> as such
> [often with negative] in the exact sense of the word: it is possible to stay overnight here although there is no guest house as such.

另外统一下阶段的表述叭（

File: article/ECMA-TC39/Stage-3-Proposal-Reviews.md (hunk 2 above)
Comment (PassionPenguin) [reply article/ECMA-TC39/Stage-3-Proposal-Reviews.md:16]:
```suggestion
评审人员的反馈将会在提案库的 GitHub 议题（issue）中被追踪（[示例](https://github.com/tc39/proposal-regexp-unicode-property-escapes/issues/3)）。在会议开始之前，给评审人员发一封提醒邮件可能也是有帮助的。
```

**Review by PassionPenguin — CHANGES_REQUESTED** [review]

File: article/ECMA-TC39/Stage-3-Proposal-Reviews.md (hunk 3 above)
Comment (KimYangOfCat) [reply article/ECMA-TC39/Stage-3-Proposal-Reviews.md:17]: 那可能我看错了吧

**Comment by Usualminds** [comment]
@lsvih 校对意见已合入

**Review by lsvih — APPROVED** [review]

**Comment by Usualminds** [comment]
@lsvih 已发布 https://juejin.cn/post/6967535392484491301/

**Pull request merged**